client/src/App.js

```diff
@@ -3,7 +3,7 @@ import GlobalStyles from './GlobalStyles';
 import { ThemeProvider } from 'emotion-theming';
 import darkTheme from './themes/darkTheme';
 import Header from './components/Header';
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
```

Comment:
This has something to do with button header? Maybe something like this is better in a new branch called cleanup